build/external/jshint/jshint.js

```diff
@@ -58503,7 +58505,8 @@ var warnings = {
 	W117: i18n._("\"{a}\" is not defined. Make sure you're spelling it correctly " +
 		"and that you declared it."),
 	W118: i18n._("'{a}' is only available in Mozilla JavaScript extensions (use moz option)."),
-	W119: i18n._("'{a}' is only available in ES6 (use esnext option)."),
+	W119: i18n._("'{a}' is only available in ES6 which is not supported by " + 
```

Comment:
Oops. Sorry bout that. I do not have access to my currently but once I do I'll fix this and let you know. Sorry for the goof-up.